src/repr/src/scalar.rs

```diff
@@ -1678,11 +1695,11 @@ impl<'a> ScalarType {
 lazy_static! {
     static ref EMPTY_ARRAY_ROW: Row = unsafe {
         Row::from_bytes_unchecked(vec![
-            22, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            23, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
These must have been a tricky error to figure out. This is because the type ids of `Array` and `List` increased by one when adding `Uint32`? I wonder if there's a better way than hard-coding the type ids here.